plugins/extraction/miniml.mli

```diff
@@ -71,7 +71,9 @@ type inductive_kind =
 
 type ml_ind_packet = {
   ip_typename : Id.t;
+  ip_typename_ref : GlobRef.t;
   ip_consnames : Id.t array;
+  ip_consnames_ref : GlobRef.t array;
```

Comment:
seems like we could do just
```suggestion
  ip_ind : inductive;
  ip_consnames : Id.t array;
```
since the constructor data is canonically derivable from it

Reply:
The whole point of this PR is to make the GlobRef.t type opaque to replace it with an abstract type of extraction names.

Reply:
ok then